app/scripts/controllers/membership.js

```diff
@@ -364,5 +363,25 @@ angular
                 }
               });
             });
+
+
+          DataService
+            .list('serviceaccounts', requestContext)
+            .then(function(resp) {
+              var serviceAccounts = _.map(resp.by('metadata.name'), function(sa) {
+                return sa.metadata.name;
+              });
```

Comment:
> Prefer to eliminate it from the dropdown?

I think so to avoid confusion. Probably want to sort as well.

```js
var serviceAccountNames = _.keys(resp.by('metadata.name')).sort();
```